backend/src/Connection.ts

```diff
@@ -25,7 +24,7 @@ export default class Connection extends EventEmitter {
 
   set({ host = IP_PRIVATE, restriction = IP_OPEN, failover = true, ...connection }: IConnection, setCLI?: boolean) {
     this.params = { host, restriction, failover, ...connection }
-    Logger.info('SET CONNECTION', { params: this.params })
+    d('SET CONNECTION', { params: this.params })
```

Comment:
Too much logging - would be good to log just the changes, but not a priority now.